adrf/viewsets.py

```diff
@@ -155,14 +155,13 @@ def view_is_async(cls):
         """
         Checks whether any viewset methods are coroutines.
         """
-        result = [
+        return any(
```

Comment:
This change is just a minor optimization so that it can use short-circuit logic when the first coroutine is found.